OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitOnlyOnYoutubeFragment.java

```diff
@@ -56,4 +80,13 @@ public void onActivityCreated(Bundle savedInstanceState) {
     public void run() {
         ViewPagerDownloadManager.instance.done(this, true);
     }
+
+    private void initializeYoutubePlayer() {
+        String url = ((VideoBlockModel) unit).getData().encodedVideos.youtube.url;
+        String videoId = url.split("v=")[1];
+        youTubePlayerFragment = YouTubeMediaPlayerSupportFragment.newInstance(youtubeApiKey, videoId);
+
+        FragmentTransaction transaction = getChildFragmentManager().beginTransaction();
+        transaction.replace(R.id.youtube_fragment, youTubePlayerFragment).commit();
```

Comment:
here you can merge declaration and initialization like this `getChildFragmentManager().beginTransaction().replace(R.id.youtube_fragment, youTubePlayerFragment).commit()`